text/levenshtein_distance.ts

```diff
@@ -6,6 +6,10 @@
  * {@link https://en.wikipedia.org/wiki/Levenshtein_distance | Levenshtein distance}
  * between two strings.
  *
+ * > ![NOTE]
+ * > The complexity of this function is O(m * n), where m and n are the lengths
+ * > of the two strings.
```

Comment:
Maybe let's also mention that it is recommended to limit/validate the length of the input if you accept arbitrary input from the users